test/smoketest/main.py

```diff
@@ -778,12 +777,15 @@ def county_audit(ac, county_id):
             r = test_endpoint_get(ac, ac.state_s, "/dos-dashboard", show=False)
             discrepancies = ""
             contest_discrepancies = r.json().get('discrepancy_count', {})
-            for contest_id, d in contest_discrepancies.iteritems():
+            for contest_id, d in contest_discrepancies.items():
                 discrepancies += "%s %2d %2d %2d %2d %2d  " % (contest_id, d["2"], d["1"], d["0"], d["-1"], d["-2"])
             logging.debug(discrepancies)
 
         total_audited = county_dashboard['audited_ballot_count']
 
+        if selected.get(i) == None:
```

Comment:
Sorry about the sloppy commit. When you request a county_audit twice, the response from the server will be an error which resulted in a KeyError on selected[0]['db_id']. I've changed this so it handles the error, both the response error and just in case, the content not fitting expectations.